slurm/2_ApplicationSpecific/VASP-GPU-podman/sample_SLURM_vasp_std.bash

```diff
@@ -10,16 +10,14 @@
 #SBATCH --gpus-per-node=1
 #SBATCH --exclusive
 #SBATCH --output=%j.out
-# 30 minutes  walltime
+# 1 hour walltime
 #SBATCH --time=01:00:00
 
-# directory in which to write job output file [job_id].out:
-#SBATCH --chdir="/projects/academic/tonykew/tonykew/VASP"
+# Path to the vasp support scipts (you shouldn't need change this)
+bin_dir="/projects/academic/ccrgroup/VASP/bin"
 
-##############################################################################
-bin_dir="/vscratch/grp-tonykew/VASP/bin"
-data_dir="/projects/academic/tonykew/tonykew/VASP/data"
-##############################################################################
+# Configure the VASP data directory here:
+data_dir="/projects/academic/ccrgroup/ccruser/VASP/data"
```

Comment:
The format for username throughout our docs is `[CCRusername]` or you can use `$USER`